### DISCUSSION TITLE
Cannot Access dcm4chee-arc/aets/DCM4CHEE/rs

### DISCUSSION THREAD

**Discussion by victormongi** [dcm4che/dcm4chee-arc-light#3786] [Q&A] [answered]
UI working
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/5617746/187847095-86d06d7c-b542-4c53-8ce4-689ca38c5504.png">

Wado working
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/5617746/187847296-d8207959-8f6e-4ff6-b252-ab142b234927.png">

aets/DCM4CHEE/rs not working
<img width="1635" alt="image" src="https://user-images.githubusercontent.com/5617746/187847351-45ec7227-9c38-4900-8292-dd71c286eb51.png">

docker logs:
<img width="1787" alt="image" src="https://user-images.githubusercontent.com/5617746/187847805-77d853d7-8dcd-406d-b638-f4ff42b72e69.png">

Im using docker file
image: dcm4che/slapd-dcm4chee:2.6.2-27.0
image: dcm4che/postgres-dcm4chee:14.4-27
image: dcm4che/dcm4chee-arc-psql:5.27.0

What could be the problem? or should I use certain version? thank you.

**Comment by gunterze** [comment, marked as answer]
There is no RESTful service with Target URL `dcm4chee-arc/aets/{aet}/rs`. s. https://github.com/dcm4che/dcm4chee-arc-light/wiki/RESTful-Services
